pyaedt/filtersolutions_core/multiple_bands_table.py

```diff
@@ -0,0 +1,167 @@
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from ctypes import POINTER
+from ctypes import byref
+from ctypes import c_char_p
+from ctypes import c_int
+from ctypes import create_string_buffer
+
+import pyaedt
+
+
+class MultipleBandsTable:
+    """Manipulates access to the entries of multiple bands table.
+
+    This class allows you to enter, edit or remove the entries of multiple bands table.
+    The table includes the lower and upper frequencies of the bands.
+    To access the multiple bands table, use the ``multiple_bands_table`` attribute of the ``FilterSolutions`` class.
+    A valid multiple bands table must include at both lower and upper frequencies for each band.
+    """
+
+    def __init__(self):
+        self._dll = pyaedt.filtersolutions_core._dll_interface()._dll
+        self._dll_interface = pyaedt.filtersolutions_core._dll_interface()
+        self._define_multiple_bands_dll_functions()
+
+    def _define_multiple_bands_dll_functions(self):
+        """Define C++ API DLL functions."""
+        self._dll.getMultipleBandsTableRowCount.argtype = POINTER(c_int)
+        self._dll.getMultipleBandsTableRowCount.restype = c_int
+
+        self._dll.getMultipleBandsTableRow.argtypes = [c_int, c_char_p, c_char_p, c_int]
+        self._dll.getMultipleBandsTableRow.restype = c_int
+
+        self._dll.updateMultipleBandsTableRow.argtypes = [c_int, c_char_p, c_char_p]
+        self._dll.updateMultipleBandsTableRow.restype = c_int
+
+        self._dll.appendMultipleBandsTableRow.argtypes = [c_char_p, c_char_p]
+        self._dll.appendMultipleBandsTableRow.restype = c_int
+
+        self._dll.insertMultipleBandsTableRow.argtypes = [c_int, c_char_p, c_char_p]
+        self._dll.insertMultipleBandsTableRow.restype = c_int
+
+        self._dll.removeMultipleBandsTableRow.argtype = c_int
+        self._dll.removeMultipleBandsTableRow.restype = c_int
+
+    @property
+    def row_count(self) -> int:
+        """Retrieve the total number of rows present in the multiple bands table.
+        The default is ``2``.
+
+        Returns
+        -------
+        int
+            The current number of rows in the multiple bands table.
```

Comment:
```suggestion
        """Total number of rows present in the multiple bands table.
        The default is ``2``.

        Returns
        -------
        int
           Current number of rows in the multiple bands table.
```
Properties are described as nouns.